yogstation/code/modules/mob/living/brain/MMI.dm

```diff
@@ -6,6 +6,7 @@
 
 			brain = newbrain
 			name = "[initial(name)]: [brain.real_name]"
+			to_chat(brainmob, "(<span class='notice'>You do not remember your death, how you died, or who killed you. <a href='https://forums.yogstation.net/index.php?pages/rules/'>See rule 1.7</a>.</span>") //yogs
```

Comment:
This can be confusing ,they can remember what happened and who borged them if they were alive during the process.

```On the other hand, if the character had their brain removed while they were alive, as either a humanoid or cyborg, or had their soul taken with a soulstone, they retain all memories of their life.``` -rule 1.7

Reply:
does this sound better?
If your brain was removed after your death you don't remeber, how you died, or who killed you.